pkg/auth/auth_assisted_service_mock_test.go

```diff
@@ -81,73 +83,59 @@ func (f fakeInventory) DownloadMinimalInitrd(ctx context.Context, params install
 }
 
 func (f fakeInventory) DownloadClusterFiles(ctx context.Context, params installer.DownloadClusterFilesParams) middleware.Responder {
-	file, err := ioutil.TempFile("/tmp", "test.file")
+	file, err := os.CreateTemp("", "test")
```

Comment:
Why are we even still creating this file. I don't see it used anywhere.

Reply:
Removed